server/mediaserver_test.go

```diff
@@ -177,6 +177,27 @@ func TestSelectOrchestrator(t *testing.T) {
 	assert.Equal(expSessionID, sess.PMSessionID)
 }
 
+func TestCreateRTMPStreamHandlerCap(t *testing.T) {
+	s := &LivepeerServer{
+		connectionLock:  &sync.RWMutex{},
+		rtmpConnections: make(map[core.ManifestID]*rtmpConnection),
+	}
+	createSid := createRTMPStreamIDHandler(s)
```

Comment:
Maybe add some comments indicating the cases that are being tested, similar to what we have for the orchestrator?

Reply:
added some comments